plugins/azure_sentinel/plugin.spec.yaml

```diff
@@ -0,0 +1,808 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: azure_sentinel
+title: Azure Sentinel Plugin
+vendor: rapid7
+support: community
+supported_versions: [2021-04-01]
```

Comment:
can you please update it to 2022-03-21 may be?

Reply:
The thing is that the Sentinel documentation that I worked with indicates 2021-04-01, however I checked the documentation today and now it indicates `API Version: 2021-10-01-preview` [LINK](https://docs.microsoft.com/en-us/rest/api/securityinsights/stable/incident-comments/create-or-update#clouderror)